android/src/main/java/com/swmansion/rnscreens/ScreenWindowTraits.kt

```diff
@@ -225,7 +225,11 @@ object ScreenWindowTraits {
         val window = activity.window
 
         val screenForNavBarTranslucent = findScreenForTrait(screen, WindowTraits.NAVIGATION_BAR_TRANSLUCENT)
-        val translucent = screenForNavBarTranslucent?.isNavigationBarTranslucent ?: false
+        val translucent = screenForNavBarTranslucent?.isNavigationBarTranslucent
```

Comment:
Actually it can be collapsed into one line:

`val translucent = screenForNavBarTranslucent?.isNavigationBarTranslucent ?: return`

Let me know if you are okay with these changes and I'll rework the code

Reply:
Wouldn't it return when `translucent` is `false` too then?

Reply:
I think safe-operator + elvis operator will execute "else" only if predicate is null (not false)

At least Android studio offers such refacoring + according to the docs it looks like it's indeed truth: https://kotlinlang.org/docs/null-safety.html#elvis-operator

Reply:
Ok so if you could check if it really works that way (seems counterintuitive to me 😅 ) than I'm okay with it.

Reply:
@WoLewicki I double checked and it works as expected - false/trues passed down, but `null` is filtered out 👍 

Below is the image from Android Studio that offered refactoring 😅 

<img width="708" alt="image" src="https://github.com/user-attachments/assets/620af951-6e40-4f61-a5b5-66f1a694c77f">